packages/components/src/custom-select-control/style.scss

```diff
@@ -41,18 +41,13 @@
 		display: none;
 	}
 
-	// Block UI appearance.
-	border: $border-width solid $gray-900;
-	background-color: $white;
-	border-radius: $radius-block-ui;
 	outline: none;
 	transition: none;
-
 	max-height: 400px;
-	min-width: 100%;
+	min-width: 130px;
```

Comment:
Why do we need a hardcoded px value here?

Reply:
I don't think we need it any more — I added it when I was calculating width based on the button instead of the container, where it seemed like a minimum width might be helpful. Now that we're calculating from the container, I think `width` makes more sense 👍